daemon/endpoint.go

```diff
@@ -552,7 +570,7 @@ func (d *Daemon) UpdateSecLabels(id string, add, del labels.Labels) middleware.R
 	d.containersMU.RUnlock()
 
 	if !containerFound {
-		return NewPutEndpointIDLabelsNotFound()
+		log.Infof("NewPutEndpointIDLabelsNotFound %v %v container not found", id, ep)
```

Comment:
and to answer the second part of the question, we do not exit now since it is the exact case for mesos.